ufo/static/userAddForm.html

```diff
@@ -113,6 +113,7 @@
         this.querySelector('#' + this.addFlowTextObj.id).submit();
       },
       submitPostForm: function() {
+        this.setUsersFromCheckboxes();
```

Comment:
nit: When I read this line here, I feel I need more context on what user we are setting.  So, how about the name below?
```javascript
this.setSelectedUsersFromCheckboxes();
```